docs/streams/developer-guide/write-streams.html

```diff
@@ -151,7 +151,7 @@ <h2>Using Kafka Streams within your application code<a class="headerlink" href="
         For more information, see <a class="reference internal" href="../architecture.html#streams_architecture_tasks"><span class="std std-ref">Stream Partitions and Tasks</span></a> and <a class="reference internal" href="../architecture.html#streams_architecture_threads"><span class="std std-ref">Threading Model</span></a>.</p>
       <p>To catch any unexpected exceptions, you can set an <code class="docutils literal"><span class="pre">java.lang.Thread.UncaughtExceptionHandler</span></code> before you start the
         application.  This handler is called whenever a stream thread is terminated by an unexpected exception:</p>
-        <pre class="line-numbers"><code class="language-java">// Java 8+, using lambda expressions
+        <pre class="line-numbers"><code class="language-java">// Java 11+, using lambda expressions
 streams.setUncaughtExceptionHandler((Thread thread, Throwable throwable) -&gt; {
```

Comment:
I think we should remove this comment from the code snippet. If we do, for proper formatting, we need to move this line I add the comment on, to the line above:
```
        <pre class="line-numbers"><code class="language-java">streams.setUncaughtExceptionHandler((Thread thread, Throwable throwable) -&gt; {
```